docs/pages/includes/plugins/rbac-with-friendly-name.mdx

```diff
@@ -24,6 +24,12 @@ spec:
       - resources: ['access_list']
         verbs: ['list', 'read']
 
+      # Optional: To display logins permitted by roles, the plugin also needs
+      # permission to read the role resource. This is currently only supported
+      # for a subset of plugins.
```

Comment:
Since we only include this partial for a subset of plugins, would it make sense to remove the second sentence of this comment?

Reply:
That's okay with me